diff --git a/.tmux.conf b/.tmux.conf
index 348c246..8e6a800 100644
--- a/.tmux.conf
+++ b/.tmux.conf
@@ -21,7 +21,7 @@ bind C-a send-prefix -2
 set -q -g status-utf8 on                  # expect UTF-8 (tmux < 2.2)
 setw -q -g utf8 on
 
-set -g history-limit 5000                 # boost history
+set -g history-limit 50000                 # boost history
 
 # edit configuration
 bind e new-window -n '~/.tmux.conf.local' "sh -c '\${EDITOR:-vim} ~/.tmux.conf.local && tmux source ~/.tmux.conf && tmux display \"~/.tmux.conf sourced\"'"
